pkg/flare/config_check_test.go

```diff
@@ -0,0 +1,92 @@
+package flare
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type ConfigType string
+
+const (
+	MetricsConfig ConfigType = "metrics"
+	LogsConfig    ConfigType = "logs"
+)
+
+func TestReportingDisabledInfo(t *testing.T) {
```

Comment:
```suggestion
func TestContainerExclusionRulesInfo(t *testing.T) {
```

Reply:
Fixed. I forgot to change this while addressing the review comments.